### DISCUSSION TITLE
Cancel Inventory Click

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#708] [Help] [answered]
The InventoryClickEvent doesn't seem to be cancellable, any way to still do it? I suppose I could set the items back to their original slot, but would that mean I'd have to do that for each click type? Thanks in advance.

**Comment by MrlnHi** [comment, marked as answer]
If you want to cancel an inventory click, you can listen to the `PreInventoryClickEvent` and cancel that
